modules/microshift-disabling-lvms-csi-driver.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+// * microshift_configuring/microshift-disable-lvms-csi-provider-csi-snapshot.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-lvms-csi-driver_{context}"]
+= Disabling deployments that run the CSI driver implementations
+
+Before you disable the CSI driver implementation pods, you must manually stop any user workloads that have attached volumes. Use the following procedure to disable installation of the CSI implementation pods.
+
+[NOTE]
+====
+{microshift-short} will not delete CSI driver implementation pods. You must configure {microshift-short} to disable installation of the CSI driver implementation pods during the startup process.
+====
+
+.Procedure
+
```

Comment:
```suggestion
.Procedure

.Prerequisites
- MicroShift has not be started for the first time
```

This procedure is only valid for users who are pre-defining their configuration before launching microshift. If it has been started already, LVMS will be running, and users will need to manually remove it via the uninstallation directions.

Reply:
done.